client/pages/index.js

```diff
@@ -5,6 +5,7 @@
 import React from "react";
 import Header from "../components/header/Header";
 
+
```

Comment:
Are you running prettier / eslint?
On my end it would want to delete this, as double-spacing is not recommended

Reply:
I do have Prettier, but I had it disabled while working through the errors, I have turned it back on, thanks for the catch!

Reply:
Fixed before latest push, Prettier is working and Husky will also catch things before commits.